pkg/client/ngr.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"fmt"
+	"github.com/pdok/pdok-metadata-tool/pkg/model/ngr"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+type NgrClient struct {
+	host      *url.URL
+	client    *http.Client
+	cswClient CswClient
```

Comment:
cswClient wordt niet gebruikt of geïnstantieerd. Ik denk dat als je een NgrClient maakt je impliciet ook een csw client kan maken en deze exposen. Echter is er niet 1 csw endpoint. Er zijn meerdere CSW endpoints. Dat zou betekenen dat dit een dict moet zijn of meerdere statische variablen zoals:

```
type NgrClient struct {
	host      *url.URL
	client    *http.Client
	cswClient CswClient
    cswClientInspire CswClient # voor het endpoint met alleen inspire records
}
```